.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: fpco/stack-build:lts
+    steps:
+      - checkout
+      - restore_cache:
+          name: Restore Cached Dependencies
+          keys: 
+            - stack-dependencies-{{ arch }}-{{ checksum "stack.yaml" }}-{{ checksum "fcm-client.cabal" }}
+            - stack-dependencies-{{ arch }}-{{ checksum "stack.yaml" }}
+      - run:
+          name: Resolve/Update Dependencies
+          command: stack setup
+      - run:
+          name: Build src and test dependencies
+          command: stack build --fast --test --bench --only-dependencies -j1
```

Comment:
Can you check how many cores do we have assigned for the containers and modify this parameter accordingly, please?